src/oneid/oneid-control-panel/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -374,17 +375,20 @@ export const Dashboard = () => {
           />
 
           <FormGroup sx={{ mt: 2, mb: 1 }}>
-            <FormControlLabel
-              control={
-                <Switch
-                  sx={{ mr: 2, ml: 1 }}
-                  name="requiredSameIdp"
-                  checked={formData?.requiredSameIdp || false}
-                  onChange={handleChange('requiredSameIdp')}
-                />
-              }
-              label="Required Same IDP"
-            />
+            {/* TODO: replace "change me" */}
+            <FieldWithInfo tooltipText="Change me" placement="top">
```

Comment:
```suggestion
            <FieldWithInfo tooltipText="Same IDP is a function that will return a custom request indicating whether the user has logged in using the same IDP as the previous time." placement="top">
```

also: do we have the chance to insert html link in it? This could be useful to point a specific confluence page.